pkg/storage/replica_command.go

```diff
@@ -4020,6 +4020,7 @@ func (r *Replica) adminScatter(
 		Multiplier:     2,
 		MaxRetries:     5,
 	}
+	rq.options.disableStatsBasedRebalance = args.DisableStatsBasedRebalance
```

Comment:
This is permanently changing the setting on the `ReplicateQueue`, which is definitely not what we want, since it'll disable stats based rebalancing on this node until the next time the node restarts. All we really need is to just pass in whether stats should be disabled to `processOneChange` rather than storing it as state in the `ReplicateQueue`.

Reply:
Yes, My mistake. Done.